packages/compass-components/src/components/accordion.tsx

```diff
@@ -52,12 +52,14 @@ interface AccordionProps extends React.HTMLProps<HTMLButtonElement> {
   hintText?: string;
   open?: boolean;
   setOpen?: (newValue: boolean) => void;
+  onToggle?: () => void;
```

Comment:
We already have `setOpen` here, should we use that instead of creating a new function?